action-scripts/translate_scripts/build_pot.py

```diff
@@ -67,10 +69,40 @@ def dst(self, dt):
 
 ltz = LocalTimeZone()
 
-def pre_build(ctx: BabContext):
+def pre_build(ctx: BabContext) -> Optional[BpyError]:
     print("Building POT...")
     path = Path(ctx.current_path)
     extracted_strings = {}
+
+    version_str: Optional[str] = None
+    with open(Path(ctx.current_path, "__init__.py"), 'r') as f:
+        for line in f:
+            if re.search(VERSION_REGEX, line):
+                version_str = line
+                break
+
+    if not version_str:
+        return BpyError("Can't extract version from __init__.py!")
+    
+    try:
```

Comment:
Appreciate the detailed comments. Since this one part is complicated and full try/except wrapped, maybe it could be its own vcomma2vdot function?

Another idea that is less precise in a way but could also work, something like val.repalce (",", ".").replace(" ", ""). Could split string on "version: " first too. We can be pretty confident the format there won't change. Thoughts?

Reply:
I think the current method is fine. The only assumption it makes is that `version` is a tuple with a comma at the end, everything else should handle differences gracefully. 

The only concern I have is the regex, but I think it should be ok.